amazon-cloudwatch-container-insights/container-insights-manifest-update.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
```

Comment:
do we need to add "set -e" to exit the shell when meets error?

Reply:
It makes sense to add `set -e`. Since the purpose of the commit is to sync with the latest changes in https://github.com/aws-samples/amazon-cloudwatch-container-insights, I will add the change in next sync.